### PR TITLE
NEPT-738: Add a hook to update the platform version in status report - fix(chore)

### PR DESCRIPTION
**Pull request by Fefaine**
…us report

### CODE REVIEW

**Comment by voidtek** [comment]
:+1:

File: profiles/common/modules/features/cce_basic_config/cce_basic_config.install [hunk 1]
```diff
@@ -2320,3 +2320,10 @@ function cce_basic_config_update_7215() {
     }
   }
 }
+
+/**
```
Comment (voidtek) [inline profiles/common/modules/features/cce_basic_config/cce_basic_config.install]: @Fefaine Can you add "Implements hook_update_N()." ?

**Comment by voidtek** [comment]
:+1:

**Comment by dxvargas** [comment]
:+1:

**Comment by voidtek** [comment]
:+1:

**Pull request merged**